README.md

```diff
@@ -126,7 +126,7 @@ Start the docker compose process to build and spin up the snoopForms container a
 
 ```
 
-docker compose up -d
+docker-compose up -d
```

Comment:
It's `docker compose` now: https://docs.docker.com/compose/#compose-v2-and-the-new-docker-compose-command

Reply:
Kinda hard to write readmes during that transition phase ;-) maybe we can add "(use `docker-compose` if you are on an older docker version)"

Reply:
True, thank you! I must be using an older version then 😅

Reply:
Oh, I was to fast with comitting before the second comment of yours haha - Will add that line also